web/hooks/push.go

```diff
@@ -27,13 +27,22 @@ func (wh GitHubWebHook) handlePush(payload *github.PushEvent) {
 	}
 	wh.logger.Debugf("Received push event for repository %v", repo)
 
+	if !(isDefaultBranch(payload) || repo.IsGroupRepo()) {
```

Comment:
I felt this "guard" was more difficult to read than using `!isDefaultBranch && !repo.IsGroupRepo`. I think both are correct.

Reply:
Agree, this one is harder to read. Changed to the `!isDefaultBranch && !repo.IsGroupRepo`.